openapi/crm/crm.json

```diff
@@ -226,6 +230,220 @@
           "status": "proposed"
         }
       }
+    },
+    "/{namespace}/{resourceTypeCode}": {
+      "parameters": [
+        {
+          "schema": {
+            "type": "string"
+          },
+          "name": "namespace",
+          "in": "path",
+          "required": true,
+          "description": "The namespace, PID/AGID"
+        },
+        {
+          "schema": {
+            "type": "string"
+          },
+          "name": "resourceTypeCode",
+          "in": "path",
+          "required": true,
+          "description": "The resourceTypeCode, here contacts, companies or activities."
+        }
+      ],
+      "post": {
+        "summary": "Upsert CRM Record",
+        "tags": [
+          "Resource Operations"
+        ],
+        "operationId": "UpsertCrmRecord",
+        "responses": {
+          "200": {
+            "description": "This API follows [Open API schema](https://spec.openapis.org/oas/v3.1.0#schema-object) of structure for response.",
```

Comment:
The response will not be the openapi schema for this end point. Its going to be an array of fields like we have in the `examples`

Reply:
Okay, Got it.